services/movie_session.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
+from django.db.models import QuerySet
+
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> MovieSession | QuerySet:
```

Comment:
are u sure that this method can return MovieSession obj ?

Reply:
fixed